VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/start-private-consultation/start-private-consultation.component.ts

```diff
@@ -153,7 +154,12 @@ export class StartPrivateConsultationComponent implements OnChanges {
 
     private mapParticipants(participantResponses: ParticipantResponse[]): ParticipantListItem[] {
         return participantResponses
-            .filter(p => p.hearing_role !== HearingRole.INTERPRETER && p.hearing_role !== HearingRole.MACKENZIE_FRIEND)
+            .filter(
+                p =>
+                    p.hearing_role !== HearingRole.INTERPRETER &&
+                    p.hearing_role !== HearingRole.MACKENZIE_FRIEND &&
```

Comment:
What role is Mackenzie friend? :D

Reply:
Mackenzie Friend is a hearing role - a type of individual. Still in use :)